developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -0,0 +1,118 @@
+//
+// const importObject = {
+  // imports: { imported_func: (arg) => console.log(arg) },
+// };
```

Comment:
Someone usually tells me to eliminate code commented like this... is there a specific reason we're keeping it around like this?